CHANGELOG.md

```diff
@@ -7,11 +7,34 @@ This projects adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 Up to version 8.0.0, the `major.minor` version numbers were based on the PHP version for which compatibility check support was added, with `patch` version numbers being specific to this library.
 From version 8.0.0 onwards, [Semantic Versioning](http://semver.org/) is used.
 
+**IMPORTANT**: The 8.0.0 release contains a **breaking change**. Please read the below information carefully before upgrading!
+
 
 ## [Unreleased]
 
 _Nothing yet._
 
+## [8.0.1] - 2017-08-07
+
+See all related issues and PRs in the [8.0.1 milestone].
+
+### Added
+- PHP 7.2 trailing comma in group use statement is now allowed
+- :fire: PHP 7.2 deprecated functions
+- :fire: PHP 7.2 deprecated/removed ini directives
+- :sparkles: PHP 7.2 (unset) cast handling
+
+### Changed
+- :wrench: Temporary hotfix for installed_paths (pending upstream fix)
+- :wrench: Improved clarity of the deprecated functions alternative
+- :wrench: `object` can now be used as a typehint IN PHP 7.2
```

Comment:
This one + the next line should be changed to these three entries and moved up to `Added`:
:star: `ForbiddenNames` sniff: recognize `object` as a forbidden keyword since PHP 7.2.
:star: `NewReturnTypeDeclarations` sniff: recognize generic `parent`, PHP 7.1 `iterable` and PHP 7.2 `object` return type declarations.
:star: `NewScalarTypeDeclarations` sniff: recognize PHP 7.2 `object` type declarion.